storage/src/main/java/org/apache/kafka/storage/internals/log/LogOffsetMetadata.java

```diff
@@ -28,6 +28,7 @@ public final class LogOffsetMetadata {
 
     //TODO KAFKA-14484 remove once UnifiedLog has been moved to the storage module
     private static final long UNIFIED_LOG_UNKNOWN_OFFSET = -1L;
+    public static final long REMOTE_LOG_UNKNOWN_OFFSET = -2L;
```

Comment:
We probably don't need this. The existing UNIFIED_LOG_UNKNOWN_OFFSET should be enough.